config/dev.exs

```diff
@@ -28,6 +28,8 @@ config :movie_wager_api, MovieWagerApi.Endpoint,
 # Do not include metadata nor timestamps in development logs
 config :logger, :console, format: "[$level] $message\n"
 
+config :movie_wager_api, :client, url: "http://localhost:4200"
```

Comment:
Also this should not be in dev.exs, this should be in secret. Or another config file that is NOT in source.